dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2549,19 +2549,19 @@
       "type": "string",
       "enum": [
         "Unspecified",
-        "MeterPerSecond",
-        "KilometerPerHour",
-        "KilometerPerMinute",
-        "KilometerPerSecond",
-        "MilePerHour",
+        "MetersPerSecond",
+        "KilometersPerHour",
+        "KilometersPerMinute",
+        "KilometersPerSecond",
+        "MilesPerHour",
         "Knot",
         "FootPerSecond",
         "FootPerMinute",
```

Comment:
Should these be plural too? "Knots", "FeetPerSecond", and "FeetPerMinute"?

Reply:
Updated!